.github/workflows/check.yaml

```diff
@@ -147,4 +150,26 @@ jobs:
         with:
           name: test-results
           path: testresults--all-features.json
-          
\ No newline at end of file
+
+  doctest:
+    # Run doctests separately, as nextest doesn't yet (https://github.com/nextest-rs/nextest/issues/16)
+    outputs:
+      test_results_url: ${{ steps.doctest_results.outputs.artifact-url }}
+    runs-on: ubuntu-latest
+    env:
+      RUSTDOCFLAGS: -Dwarnings
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@master
+        with: 
+          toolchain: ${{ env.RUST_TOOLCHAIN }}
+      - name: Run doc tests
+        run: |
+          cargo test --doc --all-features > doctestresults--all-features.json
```

Comment:
this will not produce JSON output
for that, you need to provide some command line params:
```sh
RUSTC_BOOTSTRAP=1 cargo test --doc -- -Z unstable-options --format json --report-time > doctestresults--all-features.json
```

Reply:
I really shouldn't be doing this while being at a conference...